csharp-basic-topics/ValueAndReferenceTypes/Tests/Test.cs

```diff
@@ -0,0 +1,77 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using ValueAndReferenceTypes;
+namespace Tests
+{
+    [TestClass]
+    public class Test
```

Comment:
This class should have a meaningful name ending with "UnitTest" instead of just "Test"